### PR TITLE
Add ROSA as a deploy target.

### PR DESCRIPTION
**Pull request by gurnben**
## Summary of Changes

This PR adds ROSA as a deployable platform via bootstrap-ks with a similar format to other bootstrap-ks targets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,5 +9,6 @@ We have automation in different subfolder of the project for each supported *KS
 * [EKS (Amazon Elastic Kubernetes Service)](eks)
 * [GKE (Google Kubernetes Engine)](gke)
 * [IKS (IBM Kubernetes Service) and ROKS (Red Hat OpenShift Kubernetes Service)](iks)
+* [ROSA (Red Hat OpenShift Service on AWS)](rosa)
```
Comment (schmidtd) [inline README.md:12]: You've broken our alphabetization here!

Comment (gurnben) [reply README.md:12]: You can thank the merge for that 😆

Comment (gurnben) [reply README.md]: Fixed!

**Review by schmidtd — APPROVED** [review]
/lgtm

**Pull request merged**